scripts/combine/setupCombineWMass.py

```diff
@@ -15,11 +16,11 @@
 parser = argparse.ArgumentParser()
 parser.add_argument("-o", "--outfolder", type=str, default="/scratch/kelong/CombineStudies")
 parser.add_argument("-i", "--inputFile", type=str, required=True)
-parser.add_argument("--qcdScale", choices=["byHelicityPt", "byPt", "byCharge", "integrated",], default="byHelicityPt", 
-        help="Decorrelation for QCDscale (additionally always by charge)")
+parser.add_argument("--qcdScale", choices=["byHelicityPtAndByPt", "byHelicityPt", "byPt", "byCharge", "integrated",], default="byHelicityPt", 
```

Comment:
Can you just make it so this allows multiple options?

Reply:
Is it worth? It was just for a test, and maybe calling a new specific one is better, since otherwise one would need to add protection in case of calling incompatible ones together